<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1028</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>630</width>
      <height>700</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Video</string>
     </attribute>
     <widget class="QGraphicsView" name="mainvideo">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>70</y>
        <width>411</width>
        <height>261</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>102</width>
        <height>65</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>822</width>
        <height>522</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="leftbutton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>510</y>
        <width>61</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/resource/left.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="rightbutton">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>510</y>
        <width>61</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/resource/right.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="downbutton">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>590</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/resource/down.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>50</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
      <property name="autoRepeatDelay">
       <number>300</number>
      </property>
     </widget>
     <widget class="QPushButton" name="upbutton">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>430</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/resource/up.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="Refrash">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>510</y>
        <width>61</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/resource/reset.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="leftround">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>470</y>
        <width>80</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/resource/leftround.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="rightround">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>470</y>
        <width>80</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/resource/rightround.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>410</y>
        <width>631</width>
        <height>16</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="Expansion">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>550</y>
        <width>80</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/resource/Expansion.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="Reduce">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>550</y>
        <width>80</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/resource/Reduce.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="RECbutton">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>510</y>
        <width>120</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:red; color:white; border-radius: 10px; padding: 2px; font: bold italic 32px</string>
      </property>
      <property name="text">
       <string>REC</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>RGB</string>
     </attribute>
     <widget class="QGraphicsView" name="RGBvideo">
      <property name="geometry">
       <rect>
        <x>17</x>
        <y>10</y>
        <width>591</width>
        <height>431</height>
       </rect>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>480</y>
        <width>631</width>
        <height>16</height>
       </rect>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QCheckBox" name="RedCheck">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>520</y>
        <width>100</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>RED</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="BlueCheck">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>520</y>
        <width>100</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>GREEN</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="GreenCheck">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>520</y>
        <width>100</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>BLUE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="RGBchannel">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>600</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>1channel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Face search</string>
     </attribute>
     <widget class="QGraphicsView" name="searchvideo">
      <property name="geometry">
       <rect>
        <x>17</x>
        <y>10</y>
        <width>591</width>
        <height>431</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="facebutton">
      <property name="geometry">
       <rect>
        <x>18</x>
        <y>550</y>
        <width>110</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">padding: 2px; background-color: green; font: bold italic 15px; color:white;</string>
      </property>
      <property name="text">
       <string>Face On</string>
      </property>
     </widget>
     <widget class="QPushButton" name="eyebutton">
      <property name="geometry">
       <rect>
        <x>138</x>
        <y>550</y>
        <width>110</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">padding: 2px; background-color: green; font: bold italic 15px; color:white;</string>
      </property>
      <property name="text">
       <string>Eye On</string>
      </property>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>480</y>
        <width>631</width>
        <height>16</height>
       </rect>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="upbody">
      <property name="geometry">
       <rect>
        <x>258</x>
        <y>550</y>
        <width>110</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">padding: 2px; background-color: green; font: bold italic 15px; color:white;</string>
      </property>
      <property name="text">
       <string>Upperbody On</string>
      </property>
     </widget>
     <widget class="QPushButton" name="downbody">
      <property name="geometry">
       <rect>
        <x>378</x>
        <y>550</y>
        <width>110</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">padding: 2px; background-color: green; font: bold italic 15px; color:white;</string>
      </property>
      <property name="text">
       <string>Lowerbody On</string>
      </property>
     </widget>
     <widget class="QPushButton" name="fullbody">
      <property name="geometry">
       <rect>
        <x>498</x>
        <y>550</y>
        <width>110</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">padding: 2px; background-color: green; font: bold italic 15px; color:white;</string>
      </property>
      <property name="text">
       <string>Fullbody On</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Detection</string>
     </attribute>
     <widget class="QGraphicsView" name="Detectionvideo">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>591</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Linedetection">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>480</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#Linedetection{padding: 2px; background-color: green; font: bold italic 15px; color:white;}</string>
      </property>
      <property name="text">
       <string>Line
Detection On</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Cornerdetection">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>570</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">padding: 2px; background-color: green; font: bold italic 15px; color:white;</string>
      </property>
      <property name="text">
       <string>Corner
Detection On</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Fouriertransform">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>570</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">padding: 2px; background-color: green; font: bold italic 15px; color:white;</string>
      </property>
      <property name="text">
       <string>Fourier
Transform On</string>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>450</y>
        <width>631</width>
        <height>16</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="Edgedetection">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>480</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">padding: 2px; background-color: green; font: bold italic 15px; color:white;</string>
      </property>
      <property name="text">
       <string>Edge
Detection On</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="RedHisto">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>60</y>
      <width>320</width>
      <height>170</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="GreenHisto">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>260</y>
      <width>320</width>
      <height>170</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="BlueHisto">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>460</y>
      <width>320</width>
      <height>170</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="connectbt">
    <property name="geometry">
     <rect>
      <x>912</x>
      <y>672</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="cntext">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>670</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Camera number</string>
    </property>
   </widget>
   <widget class="QLabel" name="Histogram">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>20</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Histogram</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="cameranumedit">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>672</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <stylestrategy>PreferDefault</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="maxLength">
     <number>10000</number>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="cursorPosition">
     <number>1</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
